nerfstudio/scripts/process_data.py

```diff
@@ -101,7 +104,10 @@ def main(self) -> None:
             )
 
         metadata_path = self.data / "metadata.json"
-        record3d_utils.record3d_to_json(copied_image_paths, metadata_path, self.output_dir, indices=idx)
+        ply_path = self.ply if hasattr(self, "ply") else None
```

Comment:
Not sure this is necessary? I think you can just pass in self.ply into `record3d_to_json`.

Reply:
oh you're right, sorry about that